StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -323,25 +328,22 @@ extension EmbeddedPaymentElement: EmbeddedFormViewControllerDelegate {
     func embeddedFormViewControllerDidCancel(_ embeddedFormViewController: EmbeddedFormViewController) {
         // If the formViewController was populated with a previous payment option don't reset
         if embeddedFormViewController.previousPaymentOption == nil {
-            self.formViewController = nil
             embeddedPaymentMethodsView.resetSelectionToLastSelection()
         }
         embeddedFormViewController.dismiss(animated: true)
     }
 
-    func embeddedFormViewControllerShouldClose(_ embeddedFormViewController: EmbeddedFormViewController) {
```

Comment:
This method name implies it would be called when you cancel, since you should close then too, so I renamed it to be clearer